newsletters/2024-05-22-newsletter.md

```diff
@@ -0,0 +1,77 @@
+# ZK Newsletter for 2024-05-22
+This release details...
+
+### Top picks
+
+### Reading materials 
+* [The near and mid-term future of improving the Ethereum network's permissionlessness and decentralization](https://vitalik.eth.limo/general/2024/05/17/decentralization.html) - post by Vitalik Buterin. (17.05.2024)
+* [Speeding Up Multi-Scalar Multiplications for Pairing-Based zkSNARKs](https://eprint.iacr.org/2024/750.pdf) - academic text by Xinxin Fan, Veronika Kuchta, Francesco Sica and Lei Xu. (20.05.2024)
```

Comment:
Top pick. Move to the bottom of this list, so we don't have the same links in top picks and a the top of reading materials.